### PR TITLE
R10/hotfixes v4.3.1

### PR DESCRIPTION
**Pull request by rajendra-ra**
Fixes of issues occured in [Testing 1](https://wiki.range.aero/en/projects/r10/systemarchitecture/dev/testing1/known-issues)

### CODE REVIEW

File: src/Vehicle/Vehicle.cc [hunk 1]
```diff
@@ -959,6 +966,11 @@ void Vehicle::_chunkedStatusTextCompleted(uint8_t compId)
             setPrearmError(messageText);
         }
     }
+    // check for autolog log downloading text
```
Comment (akshar-ra) [inline src/Vehicle/Vehicle.cc:969]: Does the trigger button wait for download text from ardupilot? From the discussion it was decided not go ahead with this,

File: src/Vehicle/Vehicle.cc [hunk 2]
```diff
@@ -1199,6 +1211,11 @@ void Vehicle::_handleComponentMessage2(mavlink_message_t& message)
                     formattedmessage = _formatTextMessage(message.compid,severity,_s);
                     // add message to list
                     _compMessages.append(formattedmessage);
```
Comment (akshar-ra) [inline src/Vehicle/Vehicle.cc:1213]: Please comment the reason for adding this new code.

File: src/Vehicle/Vehicle.cc [hunk 3]
```diff
@@ -1233,6 +1250,11 @@ void Vehicle::_handleComponentMessage2(mavlink_message_t& message)
...
+                    if(_compMessages.count()>100){
```
Comment (akshar-ra) [inline src/Vehicle/Vehicle.cc:1259]: Please comment the reason for adding this piece of code.

File: src/Vehicle/Vehicle.cc [hunk 4]
```diff
@@ -1994,23 +2021,38 @@ void Vehicle::_handleComponentsHeartbeat(mavlink_message_t& message)
...
+        _autologtriggeredcounter++;
+        _autologcounter++;
         // set status disconneted if heartbeat not received for more then 5 secs
         if(_dlbHeartbeatCount>5)
             _networkStatusFact.setRawValue(_networkStatusFact.rawValue().toInt() & ~0b1); // set dlb status disconnented
         if(_obcHeartbeatCount>5)
             _networkStatusFact.setRawValue(_networkStatusFact.rawValue().toInt() & ~0b10); // set obc status disconnented
+        if(_espHeartbeatCount>5)
+            _networkStatusFact.setRawValue(_networkStatusFact.rawValue().toInt() & ~0b100); // set esp status disconnented
+        if(_autologcounter>5){
```
Comment (akshar-ra) [inline src/Vehicle/Vehicle.cc:2043]: Please comment the reason for the code

File: src/Vehicle/Vehicle.cc (hunk 1 above)
Comment (rajendra-ra) [reply src/Vehicle/Vehicle.cc:969]: it does not wait but still read the status text to show it on UI.

**Pull request merged**